app/controllers/machine.js

```diff
@@ -60,7 +60,7 @@ module.exports = app => {
  *   An object containing the calendar object with the backup events
  *   and a list of backup events
  */
-function getBackupEvents(machine, CALENDAR_ROWS = 5)
+function getBackupEvents(machine, CALENDAR_ROWS = 5) 
```

Comment:
Should this still be hardcoded to 5 here? I'll dig into the context of this call in a bit but just want to make sure it wasn't an oversight.

Reply:
I guess I didn't think about what the default value for the parameter should be because it was already there and didn't need to be changed for the issue to be resolved, but I think it's somewhat useless to have it because it will always be overridden by the function call on line 31:  
    getBackupEvents(machine, machine.buckets.length)
since machine.buckets.length should always be defined as 0 or greater, right?

Reply:
Correct but I was thinking about in the context of a different call that might be introduced in the future. However, now that I think of it -- we are going to want to clean this whole portion of the codebase up to use an asynch call for the calendar data anyway. I'll accept it as it is! Thanks.